ql/ql/src/queries/style/AlertMessage.ql

```diff
@@ -0,0 +1,197 @@
+/**
+ * @name Alert message style violation
+ * @description An alert message that doesn't follow the style guide is harder for end users to digest.
+ *              See the style guide here: https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#alert-messages
```

Comment:
The URL doesn't render as a link in the actual alerts and I need to copy/paste the URL to go there. Can that be fixed?

Reply:
I don't know of a way to do that.